### ISSUE TITLE
root_view no longer available in phoenix_live_view

### ISSUE THREAD

**Issue by iwarshak** [appsignal/appsignal-elixir-phoenix#15]
as of https://github.com/phoenixframework/phoenix_live_view/commit/8bb6f44554f22bf580048e20562b62dd6b26e2b5 `root_view` is now private

```
==> appsignal_phoenix
Compiling 11 files (.ex)

== Compilation error in file lib/appsignal_phoenix/metadata.ex ==
** (CompileError) lib/appsignal_phoenix/metadata.ex:25: unknown key :root_view for struct Phoenix.LiveView.Socket
    lib/appsignal_phoenix/metadata.ex:25: (module)
```

**Comment by bbustard** [comment]
Having the same issue after upgrading to Elixir 1.12.0

**Comment by jeffkreeftmeijer** [comment]
Hey @iwarshak,

Thanks for opening this issue. I’ve just released version 2.0.7, which includes a patch for this. Could you try that and let me know if that resolves the issue for you?

**Comment by iwarshak** [comment]
@jeffkreeftmeijer so far, so good. thanks!
